app/assets/javascripts/discourse/app/components/sidebar/user/section-link.js

```diff
@@ -23,7 +24,23 @@ export default class SectionLink {
 
   @bind
   didStartDrag(e) {
-    this.mouseY = e.screenY;
+    if (e.button === 0) {
+      this.willDrag = true;
+      setTimeout(
+        () => {
+          this.delayedStart(e);
+        },
+        isTesting() ? 0 : 300
```

Comment:
I tried delay option in drag_to but it didn't work

https://www.rubydoc.info/gems/capybara/Capybara%2FNode%2FElement:drag_to

Reply:
Seems like that's the delay inbetween elements its dragging, not the start of the click. I am actually surprised `isTesting` works here, it isn't set for system tests (that I know of, could be wrong)